src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -493,9 +493,12 @@ rgbVideoCLK = Clock
 rgbVideoColor = Color Model
 rgbVideoComponent = RGB Video
 rgbVideoCursor = Cursor
+rgbVideoCustom = Custom
 rgbVideoData = Data in %s format
 rgbVideoHeight = Height
 rgbVideoReset = Reset Behavior
+rgbVideoResolution = Resolution
+rgbVideoResolutionOption = %s x %s
```

Comment:
Instead of "x", I would suggest to use the proper multiplication sign: "%s × %s"